tests/integration/api_v3/http/blockchainAppsMetadata.test.js

```diff
@@ -72,16 +72,34 @@ describe('Blockchain applications metadata API', () => {
 	});
 
 	it('retrieves blockchain application off-chain metadata by chainID', async () => {
-		const response = await api.get(`${endpoint}?chainID=00000001`);
+		const response = await api.get(`${endpoint}?chainID=04000000`);
```

Comment:
Can we fetch and set the current `chainID` and `network` as global variables in the `beforeAll` hook and use it everywhere so that the tests do not fail when executed against any network?

Reply:
Applied.